bootstrap/templates/kubernetes/apps/network/cloudflared/app/secret.sops.yaml.j2

```diff
@@ -5,6 +5,7 @@ metadata:
   name: cloudflared-secret
 stringData:
   TUNNEL_ID: "#{ bootstrap_cloudflare.tunnel.id }#"
+  TUNNEL_TOKEN: "#{ bootstrap_cloudflare.tunnel.token }#"
```

Comment:
Does this render out fine and encrypt with sops if the value is not set?

Reply:
Sorry for the delay, was away. I verified this indeed renders and encrypts fine.